census extractor documentation (#244)
opendatasoft/ods-documentation: source/publishing_data/04_configuring_a_source/configuring_source.rst

  * * :doc:`ArcGIS<connectors/arcgis>`
    * Used for data extracted from an ArcGIS server
    * Default
  * * :doc:`Census<connectors/census>`

>     * Used for data extracted from an Census server

Reviewer: ```suggestion
    * Used for data extracted from the US Census Bureau's API
```
